test/core/test/expect.test.ts

```diff
@@ -0,0 +1,72 @@
+import { resolve } from 'node:path'
+import type { UserConfig } from 'vitest'
+import { describe, expect, expectTypeOf, test } from 'vitest'
+import { getCurrentTest } from '@vitest/runner'
+import { runVitest } from '../../test-utils'
+
+describe('expect.soft', () => {
+  const run = (config?: UserConfig) => runVitest({ root: resolve(__dirname, './fixtures/expects'), exclude: [], setupFiles: [], testNamePattern: getCurrentTest()?.name, testTimeout: 4000, ...config }, ['soft'])
```

Comment:
We don't run Vitest separetly in core tests. Core tests are regular tests that are running in Vitest.

Failing tests should be defined in `test/failing`